renaissance-harness/src/main/scala/org/renaissance/harness/ConfigParser.scala

```diff
@@ -51,11 +51,6 @@ private final class ConfigParser(tags: Map[String, String]) {
         .valueName("<class-path>!<class-name>")
         .text("Load external plugin. Can appear multiple times.")
         .action((v, c) => c.withPlugin(v))
```

Comment:
We might want to indicate that class name is optional, i.e., `<class-path>[!<class-name>]`

Reply:
Done, thanks!